unittest_pythoncode.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+import gityghasemi
+
+class Test_pythonbf(unittest.TestCase):
+    def test_totalcode(self):
+        bfcode='+++++++++++11[->+++++++77>++++++++++110>++++++++++110>++++++++++110>+++++++++99>+++++++++99>+++++++++99>+++33>++++++66>+++++++++99>+++++++++99>++++++++++110>+++++++++99>+++++++++99>++++++++++110>+++++++++99>+++++++++99<<<<<<<<<<<<<<<<<]>.>+.>+++++.>++++++.>--.>+++.>--.>-.>-.>+++++.>--.>.>++++.>--.>++++.>+++++.>--.'
+        self.assertEqual(gityghasemi.execute(bfcode), 'Mostafa Ahangarha')
+
+        bfcode='++++[->++++++++<]-[----->>++>++<<<]>>+++++++.>-.+++.<+++++.>-------.+++.<<.>>.+.+++.------.+++++++.-----.<<+.'
+        self.assertEqual(gityghasemi.execute(bfcode), 'mehrad dehbid!')
+
+        bfcode='>+++++++[<++++++++++> -]<+.print G>> ++++++++[< +++++++++>-]<+.print I>>+++++++++++++++++++++[<++++>-]< .print T>> +++++++++++[< ++++++++> -]<+.print Y>++++++++++++++++++++++++++++++++.print ''>>+++++++[< ++++++++++> -]<+.print G>> ++++++++[< +++++++++> -]<. print H>> +++++++++++++[< +++++> -]<.print A>> +++++++++++++++++++++[< ++++> -]< -.print S>> +++++++++++++++++++++++[< +++> -]<.print E>> +++++++++++[< +++++++>-]<.print M>>++++++++[<+++++++++>-]<+.print I'
+        self.assertEqual(gityghasemi.execute(bfcode), 'GITY GHASEMI')
+        
+    def test_errors(self):
+        #test unbalance bracket
+        bfcode='++++[->++++++++<]------>>++>++<<<].'
+        self.assertRaises(Exception,gityghasemi.execute,bfcode)

```

Comment:
You still don't have execute method (function) in your code. This means you are calling a non existing method from gityghasemi.py which leads to error.